### ISSUE TITLE
Feature Request: Generic Daily Notification

### ISSUE THREAD

**Issue by trymeouteh** [iSoron/uhabits#383]
Notifications are attached to each habit. But to have each habit notification go off is a lot of notifications throughout the day. To have the option to add a generic notification to go off by the app that is not linked to a habit would be handy just to remind you to fill in you habits that you did today.

**Comment by naught101** [comment]
Yeah, this is bugging me too.  It'd be good if notifications beyond the first were grouped into the first, so you'd have:

* "Practice bass"
* "Practice bass, write journal"
* "Practice bass, and 2 other habits"

Etc.
